grasa_event_locator/templates/resetPWForm.html

```diff
@@ -1,9 +1,19 @@
 {% include "header.html" %}
 
-
   <div class="form-changePW">
+	    {% if expired %}
+	     <div class="alert alert-warning" role="alert">
+          This password reset link has expired, please generate a new one <a href="/resetPW.html">here.</a>
```

Comment:
Might I suggest the following?

```suggestion
          This password reset link has expired, please generate a new one <a href="{{ reverse("change_password") }}">here.</a>
```

I haven't tested this, it might be this Jinja syntax or instead with `{% %}`. I think this works though.

Reply:
Yup, honestly this would be more of a mySQL thing that requires scheduling (a cron job or something).

The reverse method doesn't work in a template, I recall this was something to do with parenthesis not being allowed in Django's template syntax. Writing reverse('change_password') to a variable and trying to make the template link to that also doesn't work, the link tries to go to itself.